src/common.js

```diff
@@ -409,3 +409,9 @@ function _register(name, implementation, wrapOptions) {
   }
 }
 exports.register = _register;
+
+// Expose the path to the NodeJS binary to utilities
+var isElectron = Boolean(process.versions.electron);
+if (!isElectron) {
+  exports.execPath = process.execPath;
```

Comment:
We'll just wait on #534. I'm not convinced if that's the best approach, since it's something that can easily be implemented by users of our module.